src/Core/Usecase/Form/VerifyFormLoaded.php

```diff
@@ -42,6 +54,34 @@ protected function verifyFormExists()
 		$this->verifyEntityLoaded($form, $this->identifiers);
 	}
 
+	/**
+	 * Checks that the form exists.
+	 * @param  Data $input
+	 * @return void
+	 */
+	protected function verifyFormDoesNoExistInContactPostState()
```

Comment:
this should probably be in a different trait, I think?

Reply:
No I think it can live here.